libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -350,9 +360,10 @@ - (BOOL)validateLaunchState:(NSError **)launchStateError
     BOOL validInputs = YES;
     NSMutableArray *launchStateErrorMessages = [NSMutableArray array];
     
-    // If an app config has been specified, set values from that first.
-    if (self.appConfig != nil) {
-        [self configureWithAppConfig];
+    // Attempt to load a default configuration, if one has not been configured.
+    if (_appConfig == nil) {
```

Comment:
This allows for the default configuration to be loaded, so apps don't have to specify that explicitly in code, if they have those file configurations.